include/benchmark/benchmark.h

```diff
@@ -1358,13 +1360,16 @@ class BenchmarkReporter {
 
     std::string benchmark_name() const;
     BenchmarkName run_name;
-    RunType run_type;          // is this a measurement, or an aggregate?
+    RunType run_type;  // is this a measurement, or an aggregate?
```

Comment:
no need for this comment; it's a `RunType`, so we know what it might be :)

https://google.github.io/styleguide/cppguide.html#Variable_Comments

Reply:
Got it!